js/tiles.js

```diff
@@ -905,9 +905,12 @@ const door = {
         canLeaveToTheRight: function(player){
             return true;
```

Comment:
this would need to be false, too. I will create a follow-up pull request. Thanks!